packages/amplify-category-notifications/src/channel-fcm.ts

```diff
@@ -63,24 +61,23 @@ export const enable = async (context: $TSContext, successMessage: string | undef
 
   spinner.start('Enabling FCM channel.');
   try {
-    const data = context.exeInfo.pinpointClient.updateGcmChannel(params).promise();
+    const data = await context.exeInfo.pinpointClient.updateGcmChannel(params).promise();
```

Comment:
didn't linter catch this for some reason ?

Reply:
pinpoint client isn't typed. I found this while trying to add types to `exeInfo`. The linter couldn't catch this because it doesn't know the type of `pinpointClient`.

Reply:
But to better answer the question, yes. The typescript ESLint plugin will flag unhandled promises.